src/features/game/events/landExpansion/plant.ts

```diff
@@ -77,10 +77,8 @@ export const getCropTime = (
   crop: CropName,
   inventory: Inventory,
   collectibles: Collectibles,
-  bumpkin: Bumpkin
+  bumpkin: Bumpkin | undefined
```

Comment:
Rather than doing this here, I think we should just do a bumpkin check at the beginning of this function and the `plant` action. If no bumpkin is found then we just return an error.

```
  if (!bumpkin) {
    throw new Error("You do not have a Bumpkin");
  }
```